tests/testthat/test-renderer1-geom-abline.R

```diff
@@ -6,10 +6,11 @@ p <- qplot(wt, mpg, data = mtcars) +
 info <- animint2HTML(list(p = p))
 
 tsv.file <- file.path("animint-htmltest", "geom2_abline_p_chunk1.tsv")
-tsv.data <- read.table(tsv.file, header=TRUE)
+tsv.data <- read.table(tsv.file, header=TRUE, comment.char = "")
 
 test_that("columns of abline tsv", {
-  expected.names <- sort(c("PANEL", "x", "xend", "y", "yend"))
+  expected.names <- sort(c("PANEL", "x", "xend", "y", "yend",
+                           "colour", "size", "linetype", "alpha"))
```

Comment:
@faizan-khan-iit again can you please revert this test case? The tsv file should not contain any constant columns.